Jenkins_jobs/CreateNewNode.groovy

```diff
@@ -70,7 +142,7 @@ node {
                    (Jenkins.getInstance().getComputer(newMachineName)).connect(false);
                 }
 
-                println "Machine ${newMachineName} was added with following labels ${newMachineLabels}";
+                println "Machine:'${newMachineName}'\nlabels: '${newMachineLabels}'\nremote root directory: '${remoteFS}'";
```

Comment:
Maybe add separators before and after the print statement to make it easier to separate different machines apart in the output logs